robot_radius : 0.45
num_neighbours : 4  